DuckDuckGo/AIChat/UserScript/AIChatUserScript.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  AIChatUserScript.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Common
+import UserScript
+
+final class AIChatUserScript: NSObject, Subfeature {
+
+    enum MessageNames: String, CaseIterable {
+        case openSettings
+        case getUserValues
+    }
+
+    private let handler: AIChatUserScriptHandling
+    public let featureName: String = "aiChat"
+    var broker: UserScriptMessageBroker?
```

Comment:
This one should be weak.  I've seen this thing holding refs before on iOS and macOS.  We need to eventually (someday, if ever) revise how this broker thing is implemented.